packages/vaadin-icon/theme/material/vaadin-icon-styles.js

```diff
@@ -4,8 +4,8 @@ registerStyles(
   'vaadin-icon',
   css`
     :host {
-      width: 18px;
-      height: 18px;
+      width: 24px;
```

Comment:
Aligned with https://github.com/PolymerElements/iron-icon/blob/master/iron-icon.js#L101-L102 (the visual tests were breaking with 18px defaults)